src/arbeidssøkerskjema/steg/1-Spørsmål.tsx

```diff
@@ -121,9 +121,13 @@ const Spørsmål: FC<any> = ({ ident }) => {
               onChange={settJaNeiSpørsmål}
               valgtSvar={arbeidssøker.kanBegynneInnenEnUke?.verdi}
             />
+            {arbeidssøker.kanBegynneInnenEnUke?.svarid === ESvar.NEI && (
+              <Alert size="small" variant={'warning'} inline>
+                <LocaleTekst tekst={'arbeidssøker.alert.senestEnUke'} />
+              </Alert>
+            )}
```

Comment:
Usikker på hvor denne filen egentlig brukes, så vet ikke om det skal legges til her også 😅

Reply:
Tror denne er i bruk for arbeidssøkerskjemaet vårt: 
https://familie.dev.nav.no/familie/alene-med-barn/soknad/arbeidssoker

Reply:
Men vet ikke om det gir mening å snakke om rett til overgangsstønad her 😬